test/src/device-request-prompt.spec.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2023 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import expect from 'expect';
+import {TimeoutError} from 'puppeteer';
+
+import {launch} from './mocha-utils.js';
+
+describe('device request prompt', function () {
+  let testState: Awaited<ReturnType<typeof launch>>;
+
+  beforeEach(async () => {
+    testState = await launch({
+      args: ['--enable-features=WebBluetoothNewPermissionsBackend'],
+    });
+  });
+
+  afterEach(async () => {
+    await testState.close();
+  });
+
+  it('does not crash', async () => {
+    const {page, server} = testState;
+
+    await page.goto(server.EMPTY_PAGE);
+    await page.waitForNetworkIdle();
+
+    await expect(page.waitForDevicePrompt()).rejects.toThrow(TimeoutError);
```

Comment:
Please leave a comment ref to Bug.
Also if possible provided a lower time for timeout as currently the timeout is `30_000` and the test will always fail with a Timeout from Mocha (Mocha timeout is set to `10_000`)

Reply:
> Please leave a comment ref to Bug.

Done

Reply:
The test seems to be running quite fast:

```
  device request prompt
    ✔ does not crash (40ms)
```

Are you sure this is needed?

Reply:
The default timeout from mocha is 2s (2000ms). I'll set it to 1s.